### ISSUE TITLE
[hubspot] Email domain authentication

### ISSUE THREAD

**Issue by jabby09** [KlimaDAO/klimadao#2259]
The solutions team is requesting to have the hubspot-hosted email account authenticated in the DNS.

<img src="https://uploads.linear.app/2a367ab5-b2fc-475f-abfb-0a29de5f9255/ff6e80a3-f3d1-406c-a74a-300278747b7c/21952899-3990-4278-9bb7-5613edd9f980?signature=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwYXRoIjoiLzJhMzY3YWI1LWIyZmMtNDc1Zi1hYmZiLTBhMjlkZTVmOTI1NS9mZjZlODBhMy1mM2QxLTQwNmMtYTc0YS0zMDAyNzg3NDdiN2MvMjE5NTI4OTktMzk5MC00Mjc4LTliYjctNTYxM2VkZDlmOTgwIiwiaWF0IjoxNzA3ODMzMDA2LCJleHAiOjMzMjc4MzkzMDA2fQ.6QE2ARoO5logr9X_Yl9W-uh6V4-n3v-ne9_k91YIx8A " alt="Screenshot 2024-02-13 at 9.58.14 AM.png" width="1083" height="154" />

**Objective**

Connect your email sending domain to HubSpot to ensure your [marketing emails](https://knowledge.hubspot.com/marketing-email/create-marketing-emails-in-the-drag-and-drop-email-editor) comply with the authentication standards and sending policies enforced by major email inbox providers

**Tasks**

Hubspot knowledge base instructions [here](https://knowledge.hubspot.com/marketing-email/manage-email-authentication-in-hubspot).

**In summary:**
*HubSpot will guide you through this connection process by helping you set up three separate DNS record types in your DNS provider settings: DKIM, SPF, and DMARC.*

**Comment by Atmosfearful** [comment]
![image](https://github.com/KlimaDAO/klimadao/assets/88635679/518390c0-811a-4fa8-9dea-bac02a5a5b84)

I resolved something related to klimadao.finance not being fully authenticated.

It's unclear to me if I'm supposed to do something related to carbonmark.com, which I don't see in the hubspot dashboard anywhere.

**Comment by Atmosfearful** [comment]
Marking this closed until I can get more clear instructions from them
